README.md

```diff
@@ -60,17 +60,24 @@ SERVER_TOKEN=XXX
 BOARD_ID=XXX
 
 ## Running the tests
-To run the tests 
 ```bash
 $ poetry run pytest
 ```
 
-To run test coverage 
+## Running the tests
+
+To run the tests
+
 ```bash
-$ poetry run pytest --cov=report term-missing --cov=tests
+$ poetry run pytest
 ```
 
-The UI tests are in a folder called tests_e2e and can be run separately, or not at all if you want the tests to complete more quickly. 
-They rely on chrome browser and chrome driver being available and being added to the path. 
+To run test coverage
+
+```bash
+$ poetry run pytest --cov=report term-missing --cov=tests
+
+```
+The UI tests are in a folder called tests_e2e and can be run separately, or not at all if you want the tests to complete more quickly. They rely on chrome browser and chrome driver being available and being added to the path.
 
-A few tests will actually connect directly to trello with your API key and server token to create a temporary board but this will be deleted as part of the test tear down. Other tests will just use dummy board data. 
\ No newline at end of file
+A few tests will actually connect directly to trello with your API key and server token to create a temporary board but this will be deleted as part of the test tear down. Other tests will just use dummy board data.
```

Comment:
When you add something like Vagrant it's worth adding a comment to the README.md

Just something along the lines of "Or you can run the app via a VM if you install Vagrant and ...."